### ISSUE TITLE
PROJEKT-4: Möjlighet att lägga upp grundare kategori

### ISSUE THREAD

**Issue by FredrikElliot** [studio27se/ehub#133]
IT-kategorin ska endast ha ett objekt under själva kategorin. Idag är detta inte möjligt

**Comment by MonaAdawi** [comment]
@FredrikElliot du kan stänga denna.

**Comment by FredrikElliot** [comment]
Hej Fredrik,

Kan du öppna nedan issue igen. Eventuellt kommer det behövas för Elnät… men inte helt säkert än. Men vill inte tappa bort den bara och vill slippa skapa en ny 😊

Mvh,
Mona Adawi

**Comment by MonaAdawi** [comment]
@FredrikElliot kan du döpa rubriken med inledningen: Projekt-4

**Comment by FredrikElliot** [comment]
@MonaAdawi 
Har ni hunnit bestämma er för om ni vill prioritera denna? Vi planerar att ta itu med den i nästa sprint om den fortfarande är önskvärd från er sida.
-- Fredrik

**Comment by MonaAdawi** [comment]
@FredrikElliot yes denna vill vi ha så snart det går. Hänvisar till vår chatt :)
![image](https://github.com/user-attachments/assets/7ead0405-4f00-49f1-8322-0fdaf2d5852f)

**Comment by FredrikElliot** [comment]
@MonaAdawi Denna är redo för att stängas, gissar jag, i och med app-releasen förra veckan.

**Comment by johulls** [comment]
Denna är testad och fungerar. Jag tog bort en nivå på pantmaskinen och gjorde en störning efter det. Mail, SMS och hemsidan ser OK ut. Vi kan stänga ärendet.
